### PR TITLE
Create tomek.html

### PR DESCRIPTION
**Pull request by tomeksokol**

### CODE REVIEW

**Comment by AlbertT96** [comment]
feature/cośtam

File: src/partials/tomek.html [hunk 1]
```diff
@@ -0,0 +1 @@
+nowy projekt
```
Comment (KamWit7) [inline src/partials/tomek.html:1]: Comment :D

Comment (KamWit7) [reply src/partials/tomek.html:1]: My reviev kamil

**Review by KamWit7 — APPROVED** [review]
review

**Pull request closed without merging**